vite.config.js

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
```

Comment:
We should add support for changing the base path via env variables, so that we can build previews 

```suggestion
  base: process.env.PUBLIC_URL || './',
  build: {
```

Please note that Vite base paths need to be either full paths, `'./'`, or `''`, so we need to update the preview CI (`.github/workflows/preview.yml`) so that it uses the full URL instead of the relative path.

Reply:
Following [docs](https://vitejs.dev/guide/static-deploy.html#github-pages), I've comitted
```
base: process.env.PUBLIC_URL || '/',
```

Please note that when defining the `PUBLIC_URL` you should:
- omit the domain
- use leading and trailing slash

> e.g. `/preview/PoliNetworkOrg/TheTOLProject/branch/main/`